app/bundles/ProjectBundle/Service/ProjectEntityLoaderService.php

```diff
@@ -240,13 +249,19 @@ private function getEntityLabelColumn(string $entityType): string
      */
     private function normalizeEntityType(string $entityType): string
     {
-        return match ($entityType) {
+        // Create event with default mappings
+        $event = new EntityTypeNormalizationEvent([
             'leadlist'       => 'segment',
             'lead'           => 'contact',
             'dynamiccontent' => 'dynamicContent',
             'trigger'        => 'pointtrigger',
-            default          => $entityType,
-        };
+        ]);
+
+        // Dispatch event to allow bundles to add their mappings
+        $this->eventDispatcher->dispatch($event, ProjectEvents::ENTITY_TYPE_NORMALIZATION);
```

Comment:
This isn't necessary anymore in newer Symfony versions. Let's ditch it and simplify this code.
```suggestion
        $this->eventDispatcher->dispatch($event);
```

Reply:
Fixed.